sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/models/application-offering-change-request.dto.ts

```diff
@@ -91,31 +91,42 @@ export class InProgressOfferingChangePaginationOptionsAPIInDTO extends Paginatio
 }
 
 /**
- * Application Offering Change Request details.
+ * Application Offering Change Request details (Student).
  */
-export class ApplicationOfferingChangesAPIOutDTO {
-  id: number;
-  status: ApplicationOfferingChangeRequestStatus;
-  applicationId: number;
+export class ApplicationOfferingDetailsAPIOutDTO {
   applicationNumber: string;
   locationName: string;
-  activeOfferingId: number;
+  status: ApplicationOfferingChangeRequestStatus;
   requestedOfferingId: number;
+  activeOfferingId: number;
+  reason: string;
+}
+
+/**
+ * Application Offering Change Request details (Institution).
+ */
+export class ApplicationOfferingChangesAPIOutDTO extends ApplicationOfferingDetailsAPIOutDTO {
+  id: number;
+  applicationId: number;
   requestedOfferingDescription: string;
   requestedOfferingProgramId: number;
   requestedOfferingProgramName: string;
-  reason?: string;
   assessedNoteDescription?: string;
   studentFullName: string;
 }
 
-export class ApplicationOfferingDetailsAPIOutDTO {
-  applicationNumber: string;
-  locationName: string;
-  status: ApplicationOfferingChangeRequestStatus;
-  requestedOfferingId: number;
-  activeOfferingId: number;
-  reason: string;
+/**
+ * Application Offering Change Request details (Ministry).
+ */
+export class ApplicationOfferingDetailsReviewAPIOutDTO extends ApplicationOfferingDetailsAPIOutDTO {
+  assessedNoteDescription: string;
```

Comment:
Is this not optional(will appear only for Approved and Declined Application Offering changes)